util/hud.lua

```diff
@@ -23,7 +23,7 @@ function logistica.show_popup(playerName, text, time, atBottom, scale)
     playerHud[playerName] = nil
   end
   local hudId = player:hud_add({
-    hud_elem_type = "text",
+    type = "text",
```

Comment:
Please don't remove the `hud_elem_type` line - instead simply add the new `type = "text"` line above it. 

Removing the original line entirely will cause this code to only work with 5.9+, while Logistica is meant to support 5.4+

Reply:
I took for granted it would require exclusively one of the keys, glad it works for both.
I'll have to read about change requests, thanks for reviewing!

Reply:
FYI, 5.9+ accepts either, and if both are present, then only `type` is used. [Reference here](https://github.com/minetest/minetest/blob/5.9.0/doc/lua_api.md#hud-definition). So having both is no problem.

For versions before 5.9, since Logistica is meant to support 5.4 as min version - if the hud def doesn't have `hud_elem_type` then it may not work correctly (it says it may assume default type is 'text' but I'd rather have it explicit). And those earlier game versions won't even check the `type` field, so having both is again no problem

Reply:
I was oblivious about that, thank you for the heads up.